src/Service/CloudFormation/src/ValueObject/Stack.php

```diff
@@ -310,48 +309,4 @@ public function getTimeoutInMinutes(): ?int
     {
         return $this->TimeoutInMinutes;
     }
-
-    public function validate(): void
```

Comment:
The Stack object is only used in output. This code was never executed.

Reply:
yes, exactly. I faced the same issue when implemnting the `requestBody` split, but added a small piece of code that check if the ValueObject is usedin `input` context

Reply:
:+1: